CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master
 
+- `Anyway::Config.env_prefix` method is introduced. ([@charlie-wasp][])
```

Comment:
and don't forget to add the shortcut definition in the bottom of the file)

Reply:
Haha, that's, how it works! 😅 I was struggling with it right now!